.changeset/grumpy-bees-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-graph': minor
+---
+
+Handle subtype in property processing.
```

Comment:
This changeset note is for `legend-graph`. Subtype should not be mentioned at all. This changeset should address the extension you are adding.